assets/js/blocks/rating-filter/edit.tsx

```diff
@@ -100,14 +85,16 @@ const Edit = ( {
 			{ getInspectorControls() }
 			{
 				<div { ...blockProps }>
-					<BlockTitle
-						className="wc-block-rating-filter__title"
-						headingLevel={ headingLevel }
-						heading={ heading }
-						onChange={ ( value: string ) =>
-							setAttributes( { heading: value } )
-						}
-					/>
+					{ heading && (
```

Comment:
Since this is a new block, and users don't have a legacy `heading` attribute is this needed?